apps/barista/src/pages/page-content.ts

```diff
@@ -35,4 +35,37 @@ export class BaPageContent {
   private _data = '';
 
   constructor(private _elementRef: ElementRef<HTMLElement>) {}
+
+  ngAfterContentInit(): void {
+    const allColorItems = document.querySelectorAll('.js-ba-copybutton');
+    if (allColorItems && allColorItems.length > 0) {
+      allColorItems.forEach(colorItem => {
+        colorItem.addEventListener('click', this._handleCopyColor);
```

Comment:
Event listener should be closed when the component is destroyed

Reply:
I think when the component is destroyed, all it's child dom elements are removed from the DOM, which usually clears their event handlers (unless angular caches the components for re-rendering somewhere?). 

To avoid non angular legacy scripts, can we drop the copy color functionality for now?